Source/WebGPU/WebGPU/HardwareCapabilities.mm

```diff
@@ -81,6 +81,7 @@
     // WGPUFeatureName_TextureCompressionETC2 and WGPUFeatureName_TextureCompressionASTC are to be filled in by the caller.
 
     // FIXME: Determine if WGPUFeatureName_IndirectFirstInstance is supported.
+    features.append(WGPUFeatureName_RG11B10UfloatRenderable);
```

Comment:
https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf doesn't list MTLPixelFormatRG11B10Float; why do we think it's renderable everywhere?

Reply:
Can we mark it not supported until there is official documentation indicating that it is?

Reply:
Oh, the feature set tables have a typo in them. Sorry for the noise.